frontend/app-development/layout/PageHeader/SubHeader/SettingsModalButton/SettingsModal/components/Tabs/AboutTab/InputFields/InputFields.tsx

```
@@ -1,66 +1,134 @@
+  const appHasLanguageTranslation = appLangCodes.includes(language);
+
+  const handleOnBlur = (event: ChangeEvent<HTMLInputElement>) => {
+    const isValid = validateServiceName(event.target.value);
+    if (isValid) onSave(event.target.value, language);
   };
 
-  const validateForm = (form: AppConfigForm): Boolean => {
-    if (form.serviceName.length <= 0) {
-      setAppConfigFormErrors({ serviceName: t('settings_modal.about_tab_name_error') });
+  const validateServiceName = (newServiceName: string): Boolean => {
+    if (newServiceName.length <= 0) {
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)